README.markdown

```diff
@@ -507,6 +507,20 @@ func reticulateSplines(spline: [Double], adjustmentFactor: Double,
 }
 ```
 
+When declaring a new function, we recommend using markup to create a richly formatted 'Quick Help' describing the function, it's paremeters and it's return object. This way, other users simply have to select the option key and select the function in any context to see the information about the function with it's intended use.
+
+To create a 'Quick Help' style decleration format your function like this which can be automatically done using the hotkey OPT+CMD+/:
```

Comment:
Typo: It should say ```declaration``` instead of ```decleretion```.
Also, isn't it something missing in this sentence? It's hard for me to understand the part: "(...)style decleration format your function like this which can be automatically (...)"

Reply:
Maybe:

You can create a 'Quick Help' declaration format for your function using the hotkey `OPT+CMD+/`:

Reply:
I've updated the branch with your suggestion @dfreniche